packages/common/client-utils/package.json

```diff
@@ -98,7 +98,7 @@
 		"test:mocha:cjs": "mocha --recursive \"dist/test/mocha/**/*.spec.*js\" --exit",
 		"test:mocha:esm": "mocha --recursive \"lib/test/mocha/**/*.spec.*js\" --exit",
 		"tsc": "fluid-tsc commonjs --project ./tsconfig.cjs.json && copyfiles -f ../../../common/build/build-common/src/cjs/package.json ./dist",
-		"typetests:gen": "flub generate typetests --dir . -v --level public",
+		"typetests:gen": "flub generate typetests --dir . -v",
```

Comment:
Is this and the below config change having an impact? If not, would be nice to separate from the bump. Okay to keep.

Reply:
It's not necessary; the --level flag is deprecated in 0.46 so this squashes a warning during builds but overall it's a no-op.